pkg/http/api.go

```diff
@@ -297,15 +297,15 @@ func (a *api) onOutputBindingMessage(reqCtx *fasthttp.RequestCtx) {
 	err := a.json.Unmarshal(body, &req)
 	if err != nil {
 		msg := NewErrorResponse("ERR_INVOKE_OUTPUT_BINDING", fmt.Sprintf("can't deserialize request: %s", err))
-		respondWithError(reqCtx, 500, msg)
+		respondWithError(reqCtx, net_http.StatusInternalServerError, msg)
```

Comment:
please use `fasthttp.StatusInternalServerError` instead of net_http one because we use fasthttp server for http endpoint.